code/parse/sexp/LuaShared.cpp

```diff
@@ -0,0 +1,63 @@
+#include "parse/sexp/LuaShared.h" 
+#include "object/object.h"
+#include "ship/awacs.h"
+#include "ship/ship.h"
+#include "network/multiutil.h"
+
+
+
+/**
+ * Determine if ship visible on radar
+ * 
+ * @return 0 - not visible
+ * @return 1 - marginally targetable (jiggly on radar)
+ * @return 2 - fully targetable
+ */
+int shared_check_ship_visibility(const ship *viewed, ship *viewer)
```

Comment:
This function should be defined in the ship code since it has no inherent relation to SEXP or Lua as far as I can tell.

Reply:
Fair enough, I've moved and renamed it, and excised the new files.